ui/tests/integration/components/dashboard/quick-actions-card-test.js

```diff
@@ -94,39 +93,39 @@ module('Integration | Component | dashboard/quick-actions-card', function (hooks
   test('it should show quick action empty state if no engine is selected', async function (assert) {
     await this.renderComponent();
     assert.dom('.title').hasText('Quick actions');
-    assert.dom('[data-test-secrets-engines-select]').exists({ count: 1 });
-    assert.dom('[data-test-component="empty-state"]').exists({ count: 1 });
+    assert.dom(SELECTORS.secretsEnginesSelect).exists({ count: 1 });
+    assert.dom(SELECTORS.emptyState).exists({ count: 1 });
```

Comment:
I think asserting the empty state message and/or title have the correct text is a more honest check that the UI renders as expected